intrinsic_matrix: [[514.177765, 0.000000, 346.861136], [0.000000, 513.054629, 220.015799], [0, 0, 1]]
# r_B_C: [0.7071, 0, 0,0.7071]
# r_B_C: [1, 0, 0, 0]
# r_B_C: [1, 0, 0, 0]
r_B_C: [0.7071, -0.7071,0, 0]
t_B_C: [-0.2,0,-0.2]

r_I_B: [0.7071,0,0,0.7071] # (x,y,z,w)
t_I_B: [0,0,0]

period_ms: 1
